src/state/queries/notifications/feed.ts

```diff
@@ -81,6 +83,7 @@ export function useNotificationFeedQuery(opts?: {enabled?: boolean}) {
             queryClient,
             moderationOpts,
             fetchAdditionalData: true,
+            ungroupFollowBacks: gate('ungroup_follow_backs'),
```

Comment:
This makes the check early — before we know if it would have any effect (it would only have effect if someone actually follows you back). So we're going to lose some statistical power if we do it this way.